install_config/aggregate_logging.adoc

```diff
@@ -1874,15 +1811,66 @@ systemctl restart docker
 
 . Update the xref:fluentd-upgrade-source[Fluentd log source].
 
+[WARNING]
+====
+Restarting Fluentd on more than a dozen nodes at once will create a large load
+on the Kubernetes scheduler. Exercise caution when using the following the
+directions to restart Fluentd.
+====
+
+There are two methods for restarting Fluentd. You can restart the Fluentd on one
+node or a set of nodes, or on all nodes.
+
+The following steps demonstrate how to restart Fluentd on one node or a set of
+nodes.
+
+. List the nodes where Fluentd is running:
++
+----
+$ oc get nodes -l logging-infra-fluentd=true
+----
+
+. For each node, remove the label and turn off Fluentd:
++
+----
+$ oc label node $node logging-infra-fluentd-
+----
+
+. Verify Fluentd is off:
++
+----
+$ oc get ds -l component=fluentd
+----
+
+. For each node, restart Fluentd:
++
+----
+$ oc label node $node logging-infra-fluentd=true
+----
+
+The following steps demonstrate how to restart the Fluentd all nodes.
+
+. Turn off Fluentd on all nodes:
++
+----
+$ oc label node -l logging-infra-fluentd=true --overwrite logging-infra-fluentd=false
+----
+
+. Verify Fluentd is off:
++
+----
+$ oc get ds -l component=fluentd
+----
+
+. Restart Fluentd on all nodes:
++
+----
+$ oc label node -l logging-infra-fluentd=false --overwrite logging-infra-fluentd=true
+----
+
 [[fluentd-upgrade-source]]
 == Updating Fluentd's Log Source After a Docker Log Driver Update
```

Comment:
Since step 2 of this section, "Updating Fluentd’s Log Source After a Docker Log Driver Update" provides instructions on using `USE_JOURNAL`, should it be removed or deprecated?

Reply:
`USE_JOURNAL` is ignored.  The step is just to get rid of it, to avoid any future confusion.  A simpler way to get rid of it:
```
oc set env -n logging ds/logging-fluentd USE_JOURNAL-
```